src/openforms/js/lang/nl.json

```diff
@@ -836,6 +836,11 @@
     "description": "Errored Submissions Removal Limit help text",
     "originalDefault": "Amount of days errored submissions of this form will remain before being removed. Leave blank to use value in General Configuration."
   },
+  "HNB0l+": {
+    "defaultMessage": "The legacy document types configuration will be ignored as soon as a catalogue is selected, even if you don't select any document type in the dropdowns.",
```

Comment:
I've probably asked this before, but translations will be added later I presume?

Reply:
yep!